### PR TITLE
Public Red Coalition Flag

### PR DESCRIPTION
**Pull request by Geevies**
* Added the Martian Red Coalition flag and banner to the loadout.

by request of sycmos + has human lore member permission i think

### CODE REVIEW

**Comment by Geevies** [comment]
!review

File: code/game/objects/structures/signs.dm [hunk 1]
```diff
@@ -719,6 +719,29 @@
 	name = "Large Old World flag"
 	flag_size = 1
 
+/obj/item/flag/red_coalition
+	name = "Red Coalition flag"
```
Comment (SleepyGemmy) [inline code/game/objects/structures/signs.dm:723]:
```suggestion
	name = "\improper Red Coalition flag"
```

File: code/game/objects/structures/signs.dm [hunk 2]
```diff
@@ -719,6 +719,29 @@
...
+	desc = "A high-quality copy of an original Red Coalition banner. This variant on the standard was flown by the Zelazny arcology during the Martian World War, Zelazny's origins as a \
```
Comment (SleepyGemmy) [inline code/game/objects/structures/signs.dm:724]:
```suggestion
	desc = "A high quality copy of an original Red Coalition banner. This variant on the standard was flown by the Zelazny arcology during the Martian World War, Zelazny's origins as a \
```

File: code/game/objects/structures/signs.dm [hunk 3]
```diff
@@ -719,6 +719,29 @@
...
+	mining colony represented in the center by the alchemical symbol for iron."
+	icon_state = "coalition_flag_boxed"
+	flag_path = "redcoalition"
+
+/obj/item/flag/red_coalition/l
+	name = "Large Red Coalition flag"
```
Comment (SleepyGemmy) [inline code/game/objects/structures/signs.dm:730]:
```suggestion
	name = "large Red Coalition flag"
```

File: code/game/objects/structures/signs.dm [hunk 4]
```diff
@@ -719,6 +719,29 @@
...
+	flag_size = 1
+
+/obj/structure/sign/flag/red_coalition
+	name = "Red Coalition flag"
```
Comment (SleepyGemmy) [inline code/game/objects/structures/signs.dm:734]:
```suggestion
	name = "\improper Red Coalition flag"
```

File: code/game/objects/structures/signs.dm [hunk 5]
```diff
@@ -719,6 +719,29 @@
...
+	desc = "A high-quality copy of an original Red Coalition banner. This variant on the standard was flown by the Zelazny arcology during the Martian World War, Zelazny's origins as a \
```
Comment (SleepyGemmy) [inline code/game/objects/structures/signs.dm:735]:
```suggestion
	desc = "A high quality copy of an original Red Coalition banner. This variant on the standard was flown by the Zelazny arcology during the Martian World War, Zelazny's origins as a \
```

**Review by SleepyGemmy — APPROVED** [review]

**Review by alsoandanswer — APPROVED** [review]

**Pull request merged**